compile_proto_wrappers.sh

```diff
@@ -20,4 +20,4 @@ protoc \
   --proto_path=.. \
   --python_out=. \
   --js_out=import_style=commonjs,binary:js \
-  ../ord-schema/proto/*.proto
+  ../ord-schema/proto/*.proto --pyi_out=.
```

Comment:
please put this flag after --python_out

Reply:
fixed in https://github.com/open-reaction-database/ord-schema/pull/713/commits/0313b8dcdd5bc44586f266637464444eeeb8a300